CMU-17-356/cmu-17-356-final-project-s23-walker
# Create User & Create Co-Op

Resolves #35 
Resolves #36 

## Description
Connects create user and create co-op, so that user can register a new user and co-op
* Modified backend to give co-op a name field
* Modified frontend to make post request to `/api/users/createandjoin`
* Made email unique in user schema

## How Has This Been Tested?
* Store current signed-in user in sessionStorage
* Fill out form with new user
* Verify that new user & co-op were successfully created in MongoDB
* Verify that creating another user with the same email doesn't work

## Types of changes




## Checklist:



-   [ ] My change requires a change to the documentation, which is located at the [Wiki](https://github.com/CMU-17-356/cmu-17-356-final-project-s23-walker/wiki)

## To-Do
* Modify backend to use JWT tokens for login
* Modify frontend to track token in sessionStorage instead of current user email
